pkg/pillar/types/dpc.go

```diff
@@ -295,7 +297,7 @@ func (config *DevicePortConfig) RecordPortFailure(ifname string, errStr string)
 // DoSanitize -
 func (config *DevicePortConfig) DoSanitize(log *base.LogObject,
 	sanitizeTimePriority bool, sanitizeKey bool, key string,
-	sanitizeName, sanitizeL3Port bool) {
+	sanitizeName, sanitizeL3Port, sanitizeSharedLabels bool) {
```

Comment:
5 boolean arguments (or even 4) is hard to read at the call sites.
So I wonder if it would make sense to replace them with a SanitizeOptions struct so that the booleans can be named a the call sites.

Reply:
Makes sense, done.